### DISCUSSION TITLE
Unable to use i3wm after switching to Intel as the main GPU

### DISCUSSION THREAD

**Discussion by sashaNull** [i3/i3#4696] [Q&A] [answered]
I installed i3wm when I was using NVIDIA as the main GPU. But after switching to Intel using "sudo prime-select intel" because it was too hot and eating up all of my battery, i3wm became unusable. After uninstalling the NVIDIA drivers, now I can't use i3 even with NVIDIA, too :(. When I try to log in with i3, it automatically switches to the gnome desktop environment. Any suggestion?

**Comment by Airblader** [comment]
Look at log files. I can't tell you which ones since you gave zero details about your system.

**Comment by sashaNull** [comment, marked as answer]
So it was not directly related to GPU actually. It is because I didn't load i3 in the login manager in the first place. I'm using a laptop so I have to enable Tap to Click for Gnome Login Screen in order to change the environment.
